assignment1/src/views/Game.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+import Para from '../components/Para.vue'
+</script>
+
+<template>
+  <main>
```

Comment:
You don't need to wrap this in a main tag, typically you don't use multiple main tags on your site.